doc/admin-guide/plugins/gzip.en.rst

```diff
@@ -141,6 +141,26 @@ will leave the header intact if the client provided it.
 - For when the proxy parses responses, and the resulting compression and
   decompression is wasteful.
 
+supported-algorithms
+----------------------
+
+Provides the compression algorithms that are supported. This will allow the proxy to selectively
+support certain compressions. The default is gzip. Multiple algorthims can be selected using ',' delimiter
+
+-- To selectively support only certain compression algorithms.
+
+enforce-brotli
```

Comment:
We should rip out enforce-brotli configuration option the same can be done with multiple configs like the example below.  We might want to add an allow feature and use the ! syntax like other options, but we can add that later.

```
[s.yimg.com]
remove-accept-encoding true
enabled true
flush true
compressible-content-type text/*
disallow *.js
disallow *.css
supported-algorithms gzip

[s.yimg.com]
remove-accept-encoding true
enabled true
flush true
compressible-content-type text/*
supported-algorithms br
```